<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>664</width>
    <height>546</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>664</width>
    <height>546</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>664</width>
    <height>546</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="error_txt">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>360</y>
      <width>311</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 0, 0);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QWidget" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>661</width>
      <height>551</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <widget class="QLabel" name="img_login">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>-10</y>
       <width>331</width>
       <height>561</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">image: url(:/img_log/girls_log.jpg);
border-top-left-radius: 50px;</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap>girlslogin.png</pixmap>
     </property>
    </widget>
    <widget class="QLabel" name="back_login">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>0</y>
       <width>331</width>
       <height>545</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>331</width>
       <height>545</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>331</width>
       <height>545</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);
border-bottom-right-radius: 50px;</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="sahde_img">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>331</width>
       <height>545</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:rgba(0,0,0,50);
border-top-left-radius: 50px;</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="icon_pass">
     <property name="geometry">
      <rect>
       <x>340</x>
       <y>240</y>
       <width>31</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap>Tela_login/password.png</pixmap>
     </property>
    </widget>
    <widget class="QPushButton" name="loginButton">
     <property name="geometry">
      <rect>
       <x>420</x>
       <y>310</y>
       <width>161</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Ubuntu</family>
       <pointsize>16</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButtom#loginButton{
	background-color: qlineargradient(spread:pad, x1:0, y1:0.505682, x2:1, y2:0.477, stop:0 rgba(11, 131, 230, 219), stop:1 rgba(85, 81, 84, 226));
	color:rgba(255,255,255,210);
	border-radius: 5px; 

}

QPushButtom#loginButton:hover{
	background-color: qlineargradient(spread:pad, x1:0, y1:0.505682, x2:1, y2:0.477, stop:0 rgba(150, 123, 111, 219), stop:1 rgba(85, 81, 84, 226));
}

QPushButtom#loginButton:pressed{
	padding-left:5px;	
	padding-top:5px;
	background-color:rgba(150, 123, 111, 255);
}</string>
     </property>
     <property name="text">
      <string>LOGIN</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="user_pass">
     <property name="geometry">
      <rect>
       <x>390</x>
       <y>230</y>
       <width>251</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgba(0, 0, 0,0);
border:none;
border-bottom:2px solid rgba(46,82,101,200);
padding-bottom: 7px;
</string>
     </property>
     <property name="placeholderText">
      <string>Senha</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="user_mat_log">
     <property name="geometry">
      <rect>
       <x>390</x>
       <y>160</y>
       <width>251</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgba(0, 0, 0,0);
border:none;
border-bottom:2px solid rgba(46,82,101,200);
padding-bottom: 7px;
</string>
     </property>
     <property name="placeholderText">
      <string>Digite a sua Matrícula</string>
     </property>
    </widget>
    <widget class="QLabel" name="icon_user">
     <property name="geometry">
      <rect>
       <x>340</x>
       <y>170</y>
       <width>41</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap>Tela_login/user.png</pixmap>
     </property>
    </widget>
    <widget class="QPushButton" name="cadRequest">
     <property name="geometry">
      <rect>
       <x>350</x>
       <y>480</y>
       <width>131</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButtom#cadRequest{
	background-color: qlineargradient(spread:pad, x1:0, y1:0.505682, x2:1, y2:0.477, stop:0 rgba(11, 131, 230, 219), stop:1 rgba(85, 81, 84, 226));
	color:rgba(255,255,255,210);
	border-radius: 5px; 

}

QPushButtom#cadRequest:hover{
	background-color: qlineargradient(spread:pad, x1:0, y1:0.505682, x2:1, y2:0.477, stop:0 rgba(150, 123, 111, 219), stop:1 rgba(85, 81, 84, 226));
}

QPushButtom#cadRequest:pressed{
	padding-left:5px;	
	padding-top:5px;
	background-color:rgba(150, 123, 111, 255);
}</string>
     </property>
     <property name="text">
      <string>CADASTRAR</string>
     </property>
    </widget>
    <widget class="QLabel" name="txt_acesso">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>440</y>
       <width>171</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Não possuí acesso?</string>
     </property>
    </widget>
    <widget class="QLabel" name="LogIn">
     <property name="geometry">
      <rect>
       <x>420</x>
       <y>40</y>
       <width>141</width>
       <height>61</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>28</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Log In</string>
     </property>
    </widget>
    <widget class="QPushButton" name="resetButton">
     <property name="geometry">
      <rect>
       <x>520</x>
       <y>480</y>
       <width>121</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButtom#cadRequest{
	background-color: qlineargradient(spread:pad, x1:0, y1:0.505682, x2:1, y2:0.477, stop:0 rgba(11, 131, 230, 219), stop:1 rgba(85, 81, 84, 226));
	color:rgba(255,255,255,210);
	border-radius: 5px; 

}

QPushButtom#cadRequest:hover{
	background-color: qlineargradient(spread:pad, x1:0, y1:0.505682, x2:1, y2:0.477, stop:0 rgba(150, 123, 111, 219), stop:1 rgba(85, 81, 84, 226));
}

QPushButtom#cadRequest:pressed{
	padding-left:5px;	
	padding-top:5px;
	background-color:rgba(150, 123, 111, 255);
}</string>
     </property>
     <property name="text">
      <string>RESET</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections/>
</ui>
